sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsRequestOptions.cs

```diff
@@ -40,5 +42,10 @@ internal TextAnalyticsRequestOptions(bool includeStatistics, string modelVersion
         /// <a href="https://docs.microsoft.com/azure/cognitive-services/text-analytics/how-tos/text-analytics-how-to-sentiment-analysis#model-versioning"/>.
         /// </summary>
         public string ModelVersion { get; set; }
+
+        /// <summary>
+        /// StringIndexType
+        /// </summary>
+        public StringIndexType? StringIndexType { get; set; }
```

Comment:
So we need to keep a default value for the library, so user doesn't have to think about it.
I think it will be clearer for the user if they can see the default value of this parameter here.
Because it will always have a value, make it non-nullable

Reply:
We definitely need to work on the name (it could be in another PR) as currently it is hard to understand what it means

Reply:
could u open an issue for addressing the name?

Reply:
Could u put the link here so we can have the reference?